src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -834,14 +834,16 @@ private TokenValidationResult ValidateTokenPayload(JsonWebToken jsonWebToken, To
 
             Validators.ValidateLifetime(notBefore, expires, jsonWebToken, validationParameters);
             Validators.ValidateAudience(jsonWebToken.Audiences, jsonWebToken, validationParameters);
-            var issuer = Validators.ValidateIssuer(jsonWebToken.Issuer, jsonWebToken, validationParameters);
```

Comment:
This indicates we have a difference between JsonWebTokenHandler and JwtSecurityTokenHandler.
The value returned from ValidateIssuer is set as the Claim.Issuer in the JwtSecurityTokenHandler.

https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/1164

Reply:
Good catch!